### PR TITLE
SearchV3: Fix crash in the case of null searchSuggestions

### PR DESCRIPTION
**Pull request by cjb**
@keybase/react-hackers 

The interestingPeople search suggestions RPC returns null in the case of no matches, and you have no matches if you haven't chatted or followed anyone, and this is the case that a fresh account is in.  We end up crashing with `TypeError: Cannot read property 'map' of null`.

This PR fixes the crash by synthesizing an empty search result when suggestions is null (and rewraps a comment).

### CODE REVIEW

File: shared/actions/searchv3/index.js [hunk 1]
```diff
@@ -214,6 +216,12 @@ function* searchSuggestions<T>({payload: {actionTypeToFire, maxUsers}}: Constant
     },
   })
 
+  if (!suggestions) {
```
Comment (chrisnojima) [inline shared/actions/searchv3/index.js]: maybe better to just do yield call(...) || []

**Comment by mmaxim** [comment]
I can just make this return an empty list, would that be easier?

**Comment by mmaxim** [comment]
Go has this notion that `nil` and an empty list are essentially equivalent, so a `nil` snuck out of here in the no results case.

**Comment by chrisnojima** [comment]
no its a quick fix, we should protect ourselves

**Comment by cjb** [comment]
@chrisnojima Huh, doesn't work, seems `const suggestions = yield call(..) || []` returns `null` rather than `[]` even though suggestions would otherwise be null.

**Comment by chrisnojima** [comment]
I think you need it wrapped in extra parens. Or use let and assign it on
the next line. The fear is if there's more logic below we want it to run
and avoid extra flows like this. Better to just let it fall through

**Comment by cjb** [comment]
@chrisnojima k, this works

**Comment by cjb** [comment]
Oh!  @MarcoPolo had a fix for this in an old PR of his that he just merged in.

**Comment by cjb** [comment]
@MarcoPolo I like my fix better 'cause it leads to an empty search result screen instead of a "Securing..", this okay with you?

**Pull request merged**